scholia/app/views.py

```diff
@@ -134,6 +135,98 @@ def redirect_q(q):
     return redirect(url_for(method, q=q), code=302)
 
 
+def bioschemas_type(bioschemas_type, conforms_to):
+    """Create the basic and common content of Bioschemas.
+
+    Parameters
+    ----------
+    bioschemas_type : str
+        Bioschemas type of the thing to be represented
+    conforms_to : str
+        Profile specification for the given type
+    """
+    bs_profile = "https://bioschemas.org/profiles/"
+    return {
+        "@type": bioschemas_type,
+        "http://purl.org/dc/terms/conformsTo": {
+            "@type": "CreativeWork",
+            "@id": bs_profile + conforms_to,
+        }
+    }
+
+
+@main.route("/" + q_pattern + "/bioschemas")
+def bioschemas_for(q):
```

Comment:
@fnielsen: "Perhaps the JSON returning functions should be called show_*_json. Otherwise one tend to put an api.py module all by itself."

Reply:
I will rename the method to `show_bioschemas_json`.

Reply:
Implemented.